zerver/lib/test_runner.py

```diff
@@ -48,20 +47,36 @@ def get_test_method(test):
     # type: (TestCase) -> Callable[[], None]
     return getattr(test, test._testMethodName)
 
+# Each tuple is delay, test_name, slowness_reason
+TEST_TIMINGS = [] # type: List[Tuple[float, str, str]]
+
+
+def report_slow_tests():
```

Comment:
Can you type-annotate this?  In general, new functions should come with mypy type annotations these days (I think that's why the coverage, which includes both tests and mypy, is negative for this branch)